pkg/ipam/multipool.go

```diff
@@ -718,6 +718,25 @@ func (c *multiPoolAllocator) Dump() (map[string]string, string) {
 	return c.manager.dump(c.family)
 }
 
+func (c *multiPoolAllocator) Capacity() uint64 {
+	var capacity uint64
+	for _, pool := range c.manager.pools {
+		var p *podCIDRPool
+		switch c.family {
+		case IPv4:
+			p = pool.v4
+		case IPv6:
+			p = pool.v6
+		}
+		if p == nil {
+			continue
+		}
+		_, _, c, _, _ := p.dump()
```

Comment:
dump seems to iterate all the ips using `ForEach`.  Might be better to have a seperate function to compute just the utilization.

Reply:
Done